packages/vite-plugin-cloudflare/src/websockets.ts

```diff
@@ -0,0 +1,88 @@
+import ws from 'ws';
+import { UNKNOWN_HOST } from './shared';
+import type { Fetcher } from '@cloudflare/workers-types/experimental';
+import type { ReplaceWorkersTypes } from 'miniflare';
+import type { IncomingMessage } from 'node:http';
+import type { Duplex } from 'node:stream';
+import type * as vite from 'vite';
+
+export function handleWebSocket(
```

Comment:
I think a jsdoc comment here explaining what this does would be very helpful 🙂

Reply:
[b25120e](https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/128/commits/b25120e875d5cef8cec4b08cd3b1b4a3be7ad6a4)